### DISCUSSION TITLE
As admin be able to manage 'Preferences' of user account

### DISCUSSION THREAD

**Discussion by timbertens** [craftcms/cms#10473] [Ideas]
As an admin it would be very handy if we could manage the 'Preferences' of a user account, both 
* doing support on existing accounts 
* when creating new accounts

I know it is possible to impersonate the user and do the changes in their account, but that is an additional step.  
We want to make sure that newly created users don't have to fiddle around much on their first usage of Craft and even more importantly, we have an integration running on users of the CMS that uses the language setting to determine the communication language.

**Comment by timkelty** [comment]
@timbertens perhaps it would be even more useful to be able to select a user as a "template" when creating a user?
